src/en/help-oracle.md

```diff
@@ -100,138 +100,157 @@ Cloud Types
 
-```no-highlight
-Enter a name for your oracle cloud: myoracle
-Enter the API endpoint url for the cloud: https://compute.uscom-central-1.oraclecloud.com/
-Cloud "myoracle" successfully added
-```
+Enter a name for your oracle cloud: oracle-cloud
 
-If you're using a paid Oracle account, `uscom-central-1` is currently the only
-supported region that works with Juju out of the box. 
+Enter the API endpoint url for the cloud:
+https://compute.uscom-central-1.oraclecloud.com/
 
-For trial accounts and other regions, the endpoint needs to be retrieved from
-Oracle's Compute dashboard. 
+Cloud "oracle-cloud" successfully added
+You may bootstrap with 'juju bootstrap oracle-cloud'
+```
 
-To retrieve the endpoint, sign in to your Oracle cloud domain and click on the
-Compute tile for your domain. 
+We've called the new cloud 'oracle-cloud' and used an endpoint of
+'https://compute.uscom-central-1.oraclecloud.com/'.
 
-Alternatively, if the Compute domain tile isn't visible, use the drop-down menu
-in the top left of the dashboard to select `Cloud Account`, select the
-`Subscriptions` tab and from the 'Subscriptions Type' drop-down menu, select
-IaaS. From the list of  subscriptions that appear (Storage, Compute, Ravello
-and Container), click on the menu icon to the right of Compute and select 
-`View Details`.
+Now confirm the successful addition of the cloud:
 
-In the `Additional Information` view that appears, the REST endpoint field is
-the value Juju needs for the endpoint URL:
+```bash
+juju clouds
+```
 
-![Endpoint URL](./media/oracle_myservices-endpoint.png)
+Here is a partial output:
 
-## Credentials
+```no-highlight
+Cloud            Regions  Default          Type        Description
+.
+.
+.
+oracle                 5  uscom-central-1  oracle      Oracle Compute Cloud Service
+oracle-cloud           0                   oracle      Oracle Compute Cloud Service
+```
 
-Using Juju's interactive authentication, importing Oracle credentials into Juju
-is a simple process. You will just need the following information:
+Cloud 'oracle' is for the built-in (for pay) service and cloud 'oracle-cloud'
+is tied to your trial account.
 
-- **Username**: usually the email address for your Oracle account.
-- **Password**: the password for this specific Compute domain.
-- **Identity domain**: the ID for the domain, e.g. `a476989`.
+## Add credentials
 
-To add these details, type `juju add-credential <credential-name>
-<cloud-name>`: 
+Use the interactive `add-credential` command to add your credentials to the new
+cloud:
 
 ```bash
-juju add-credential myoracle
+juju add-credential oracle-cloud
 ```
 
-You will be asked for each detail in turn.
+Example user session:
 
 ```no-highlight
-Enter credential name: mynewcredential
+Enter credential name: oracle-cloud-creds
+
 Using auth-type "userpass".
-Enter username: javier
-Enter password: ********
-Enter identity-domain: a476989
-Credentials added for cloud myoracle.
+
```

Comment:
Javier is quite a popular name! Maybe not in Canada. Anyhow, we were just implementing the design team personas.